scalding-core/src/main/scala/com/twitter/scalding/StringUtility.scala

```diff
@@ -0,0 +1,39 @@
+package com.twitter.scalding
+
+object StringUtility {
```

Comment:
I just implemented a new one after reading your comments. Will post soon:
             mean, std, median running time (nano seconds)
Java's: (7770.3,13256.792142520753,4000.0)
Fast:    (4087.7,39296.86258100002,3000.0)

using a small epoc of 100 strings to separate, with a-z with "," only randomly generated. For instance "a,bvc,sdf,fw,f,,,"

I found no matter how large the epoc is, the std is pretty huge. Not sure why. But the median running time should reflect it's faster than java's implementation for most of the time